nixos/tests/all-tests.nix

```diff
@@ -592,7 +592,7 @@ in {
   jotta-cli = handleTest ./jotta-cli.nix {};
   k3s = handleTest ./k3s {};
   kafka = handleTest ./kafka.nix {};
-  kanboard = handleTest ./web-apps/kanboard.nix {};
+  kanboard = runTest ./web-apps/kanboard.nix ;
```

Comment:
```suggestion
  kanboard = runTest ./web-apps/kanboard.nix;
```

Reply:
It's interesting that formatter didn't catch it.

Reply:
Good point, but maybe `all-tests` hasn't been formatted.